publisher.rb

```diff
@@ -0,0 +1,47 @@
+require 'aws-sdk-sqs'
+require 'byebug'
+require_relative 'load_aws'
+
+class Publisher
+  # Note: You can store the values of the aws access keys and region list in a config file (json) 
+  # from where you can get the values into the constants.
+  ACCESS_KEY_ID = AWS["access_key_id"]
+  SECRET_ACCESS_KEY = AWS["secret_access_key"]
+  REGION = AWS["region"]
+
+  attr_accessor :sqs
+
+  def initialize
+    creds = Aws::Credentials.new(ACCESS_KEY_ID, SECRET_ACCESS_KEY)
+    @sqs = Aws::SQS::Client.new(region: REGION, credentials: creds)
+  end
+
+  # For Standard Queue
+  def create_std_q(queue_name)
+    std_queue = sqs.create_queue(queue_name: queue_name)
+  end
+
+  def publish_messages(queue_url, options=[])
+    delay_seconds = options[1].nil? ? 5 : options[1]
+    0.upto(Float::INFINITY) do |count|
+      message = { count: count, timestamp: Time.now.strftime("%GW%V%uT%H%M%S%L%z") }.to_json
+      puts "Sending message - #{message}"
+      sqs.send_message(queue_url: queue_url, message_body: message, delay_seconds: delay_seconds)
+    end
+  end
+
+  def self.run_service(options)
```

Comment:
Do we want this to be a class method?

Reply:
@glen My idea behind this was we want to run the Publisher service, more like telling the Publisher class to start its work. 
Now that you point it out, I am thinking if run_service would be more appropriate as an instance method or not.

Reply:
Changed it back.